packages/core/src/tools.ts

```diff
@@ -217,6 +218,24 @@ export namespace Tools {
 			: percentage;
 	}
 
+		/**
+	 * Truncate the labels
+	 * @export
+	 * @param {any} fullText
+	 * @param {any} truncationType
+	 * @param {any} numCharacter
+	 * @returns Truncated text
+	 */
+	export function truncateLabel(fullText, truncationType, numCharacter) {
```

Comment:
what if fullText.length is 4 and numCharacter is 10? would we still want to show the `...`?

Reply:
That's a good point. I'll show the fulltext in that case.